docs/list.md

```diff
@@ -50,6 +50,16 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **Issues**: List of [Hacktoberfest Labelled Issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc){:target="\_blank"}.
 - **Notes**: Read the [blog post](https://appwrite.io/hacktoberfest){:target="\_blank"} for more info.
 
+### **Amplication**
```

Comment:
This just needs an extra space after the header to make the bots happy and so the md -> HTML build works correctly. 🤖 😄